include/materials/material_utility.h

```diff
@@ -0,0 +1,68 @@
+#ifndef MPM_MATERIAL_UTILITY_H_
+#define MPM_MATERIAL_UTILITY_H_
+
+#include <cmath>
+
+#include "Eigen/Dense"
+
+namespace mpm {
+namespace material_utility {
+
+//! Compute mean stress p
+//! \param[in] stress Stress in Voigt notation where positive is tension
+inline const double compute_mean_p(const Eigen::Matrix<double, 6, 1>& stress);
```

Comment:
Can we include the details that p is assumed to be positive in compression here too?

Reply:
I see, ok